genestack_client/scripts/genestack_application_manager.py

```diff
@@ -1,4 +1,4 @@
-#!python
+#!/usr/bin/env python2
```

Comment:
Please revert it, it is special placeholder that will be replaced with proper shebang after installation with pip.